hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/contracts/precompile/codec/EvmEncodingFacade.java

```diff
@@ -19,7 +19,12 @@
 import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.bigIntegerTuple;
 import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.booleanTuple;
 import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.decimalsType;
+import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.getFungibleTokenInfoType;
+import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.getNonFungibleTokenInfoType;
+import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.getTokenCustomFeesType;
+import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.getTokenExpiryInfoType;
 import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.getTokenInfoType;
+import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.getTokenKeyType;
 import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.intBoolTuple;
 import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.intPairTuple;
 import static com.hedera.node.app.service.evm.store.contracts.utils.EvmParsingConstants.notSpecifiedType;
```

Comment:
Per style guide, all of these `public static final` fields should be in `CONSTANT_CASE`.  Please fix, in a different PR if needed.